test/unit/controllers/login.test.ts

```diff
@@ -1,11 +1,11 @@
-const { expect, sinon } = require('test/chai-sinon');
-const { getLogin, getLogout, postLogin, setupLoginController } = require('app/server/controllers/login.ts');
+///<reference path="../../../node_modules/@types/sinon/index.d.ts"/>
```

Comment:
What is this bit?

Reply:
No idea looks like it got pasted by mistake have removed.